client/js/commands/search.js

```diff
@@ -2,17 +2,28 @@
 
 import store from "../store";
 import {router} from "../router";
+const Msg = require("../../../src/models/msg");
 
 function input(args) {
-	router.push({
-		name: "SearchResults",
-		params: {
-			id: store.state.activeChannel.channel.id,
-		},
-		query: {
-			q: args.join(" "),
-		},
-	});
+	const {channel} = store.state.activeChannel;
+
+	if (!store.state.settings.searchEnabled) {
+		const message = new Msg({
+			type: Msg.Type.ERROR,
+			text: "Search is currently not enabled.",
+		});
+		channel.messages.push(message);
+	} else {
+		router.push({
```

Comment:
There's an existing bug here where the chat input box disappears when we use `/search`. We shouldn't use the router like this to activate the change; maybe we commit to store?